README.md

```diff
@@ -1,5 +1,5 @@
 # The Hallow
-Fabric Community mod for Hacktoberfest! Formerly known as Spooky Time.
+Fabric Community mod for Hacktoberfest! Formerly known as The Hallow.
```

Comment:
no

Reply:
nay

Reply:
In case you didn't notice it:

>>>>>>>>>
Missing fixes:
- `README.md`
>>>>>>>>>

Reply:
To be fair, I assumed this was in relation to before any changes were made at all, not changes made by this PR.

Reply:
Please review the new `README.md`.